test/testcafe/spec/ChallengeAuthenticatorEmail_spec.js

```diff
@@ -295,14 +306,15 @@ test
       .eql(`We sent an email to ${emailAddress}. Enter the verification code in the text box.`);
 
     // Verify links (switch authenticator link not present since there are no other authenticators available)
-    await t.expect(await challengeEmailPageObject.switchAuthenticatorLinkExists()).notOk();
+    await t.expect(await challengeEmailPageObject.verifyWithSomethingElseLinkExists()).notOk();
     await t.expect(await challengeEmailPageObject.signoutLinkExists()).ok();
     await t.expect(challengeEmailPageObject.getSignoutLinkText()).eql('Back to sign in');
   });
 
 test
   .requestHooks(validOTPmock)('challenge email authenticator screen has right labels', async t => {
     const challengeEmailPageObject = await setup(t);
+    await t.expect(challengeEmailPageObject.formExists()).eql(true);
```

Comment:
I think adding this to the `setup` function will make it so you won't have to call it in each test.